app/models/nic/base.rb

```diff
@@ -225,11 +225,8 @@ def children_mac_addresses
       []
     end
 
-    # we don't consider host as managed if we are in non-unattended mode
-    # in which case host managed? flag can be true but we should consider
-    # everything as unmanaged
     def host_managed?
-      host&.managed? && SETTINGS[:unattended]
+      host&.managed?
```

Comment:
Given this simplicity, should we deprecate it and suggest to inline this?

Reply:
perhaps, but please not in this PR.